Docs/user_guide/get_help.md

```diff
@@ -0,0 +1,31 @@
+# Get Help
+
+[Contact the Slicer community or commercial partners](about.html#contact-us) if you have any questions, bug reports, or enhancement requests.
+
+## I need help in using Slicer
+
+- I have a question about how to use Slicer for a particular task or I'm not sure things are working right:
+  - Type your queston into Google. There is a good chance that your question has been asked and anwered before and all questions ever asked about Slicer are publicly available and indexed by Google.
+  - Try your best to sort out the issue by reading documentation and checking error logs
+    - User and developer manual are available at https://slicer.readthedocs.io
```

Comment:
`The User Guide and Developer Guide are both available at https://slicer.readthedocs.io`
The Slicer ReadTheDocs is currently using the terminology "Guide" vs "Manual".